go-controller/pkg/ovn/egressgw.go

```diff
@@ -130,7 +125,6 @@ func (oc *Controller) getRouteInfosForNamespace(namespace string) []*externalRou
 		if namespacedName.Namespace != namespace {
 			continue
 		}
-		routeInfo.Lock()
```

Comment:
comment on these functions are no longer accurate "locked for a specific namespace"

Reply:
done